### ISSUE TITLE
[Feature request] Postgres check constraint

### ISSUE THREAD

**Issue by satishdash** [piccolo-orm/piccolo#626]
Hi,
Most of the features required for postgres integration are available I believe.
Can we add ‘check constraint’ for table models.
How do we do it today if at all i were to add a check constraint in my existing tables.

Thanks 🙏

**Comment by dantownsend** [comment]
Yes, good point - we should add that.

The best solution for now is to create a [manual migration](https://piccolo-orm.readthedocs.io/en/latest/piccolo/migrations/create.html#running-raw-sql), and add it that way.

**Comment by satishdash** [comment]
@dantownsend Any thoughts for this in the roadmap. I believe will be super useful

**Comment by dantownsend** [comment]
@satishdash I agree, this is probably the main missing feature. Just need to work out an API which fits nicely with Piccolo. If you have any ideas let me know.
